static/bidder-info/adsyield.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "http://ads-pbs.open-adsyield.com/openrtb/{{.PublisherID}}?host={{.Host}}"
+maintainer:
+  email: "engineering@project-limelight.com"
```

Comment:
@imedvedko , Thank you for your contribution to prebid server. We kindly request your assistance in verifying the email address. An email for verification is sent on `engineering@project-limelight.com`. Requesting to respond with "received" message to confirm the email address.

Reply:
got reply on verification email 👍🏼